ansible/roles_ocp_workloads/ocp4_workload_synopsys_blackduck/tasks/workload.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Setting up Synopsys Black Duck workload for user
+  debug:
+    msg: "Setting up Synopsys Black Duck workload for user ocp_username = {{ ocp_username }}"
+
+# Synopsys Black Duck workload tasks
+# -------------------------------------------------------------------------
+
+- name: Install synopsysctl CLI
+  become: true
+  unarchive:
+    src: "{{ ocp4_workload_synopsys_blackduck_ctl_url }}"
+    remote_src: yes
+    dest: /usr/bin
+    mode: 0775
+    owner: root
+    group: root
+  args:
+    creates: /usr/bin/synopsysctl
+
+- name: set the current project to the demo project to use in the j2 template
+  set_fact:
+    ocp4_workload_current_project: "{{ ocp4_workload_synopsys_blackduck_demo_project }}"
+
+- name: create {{ ocp4_workload_synopsys_blackduck_demo_project }} project for user by using a jinja2 template in the role
+  k8s:
+    state: present
+    definition: "{{ lookup('template', role_path ~ '/templates/project.j2' ) | from_yaml }}"
+
+- name: give {{ ocp4_workload_synopsys_blackduck_demo_project }} project view access to all training users
+  command: "oc adm policy add-role-to-user edit user{{ item }} -n {{ ocp4_workload_synopsys_blackduck_demo_project }}"
+  loop: "{{ range(1, num_users | int + 1, 1) | list }}"
+
+- name: Deploy insecure bank
+  k8s:
+    state: present
+    definition: "{{ lookup('file', item ) | from_yaml }}"
+    namespace: "{{ ocp4_workload_synopsys_blackduck_demo_project }}"
+  loop:
+    - ./files/insecure-bank-deployment.yml
+    - ./files/insecure-bank-service.yml
+    - ./files/insecure-bank-route.yml
+
+- name: set the current project to the blackduck project to use in the j2 template
+  set_fact:
+    ocp4_workload_current_project: "{{ ocp4_workload_synopsys_blackduck_project }}"
+
+- name: create {{ ocp4_workload_synopsys_blackduck_project }} project for user by using a jinja2 template in the role
+  k8s:
+    state: present
+    definition: "{{ lookup('template', role_path ~ '/templates/project.j2' ) | from_yaml }}"
```

Comment:
Template lookups already default to looking in the template directory under the role path, so this can be simplified to:
```
    definition: "{{ lookup('template', 'project.j2' ) | from_yaml }}"
```